docs/examples/make_pymvpa_collection.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# A little script to prepare PyMVPA demo collection based on data used in tutorials etc
+
+set -eu
+
+echo "I: adding haxby2001"
+# Haxby 2001 -- the main demo dataset
+datalad install-handle http://data.pymvpa.org/datasets/haxby2001/.git
+
+# Extracted/processed pieces from haxby2001
+echo "I: adding tutorial_data"
+datalad install-handle http://data.pymvpa.org/datasets/tutorial_data/.git
+
+echo "I: creating MNIST handle"
+datalad create-handle mnist
+(
+    cd mnist
+    git annex addurl --pathdepth -1 --fast "http://data.pymvpa.org/datasets/mnist/mnist.hdf5"
+    wget -q http://data.pymvpa.org/datasets/mnist/README.rst; git add README.rst
+    git commit -m "MNIST dataset added as a '--fast' link"
+    datalad describe \
+        --author "Yann LeCun and Corinna Cortes" \
+        --license 'CC SA 3.0' \
+        --description "Dataset of handwritten digits"# \
+        # TODO --doi "10.1109/5.726791"
+)
+
+# Forrest gump dataset, although not yet used in PyMVPA stock materials,
+# analysis scripts use PyMVPA heavily so it is a worthwhile addition
+echo "I: adding forrest_gump dataset"
+datalad install-handle http://psydata.ovgu.de/forrest_gump/.git
+
+# new plain collection
+echo "I: creating PyMVPA collection"
+datalad create-collection pymvpa_collection pymvpa # path name
+
+# add all handles to the pymvpa_collection
+echo "I: adding all handles to the PyMVPA collection"
+for h in haxby2001 tutorial_data mnist forrest_gump; do
+    datalad add-handle "$h" pymvpa
+done
+
+cd pymvpa_collection
+# XXX this should pull in the new handle meta-data, but doesn't ???
```

Comment:
Nope. It should update information from remotes (i.e. git fetch) of the collection (there are none at this point). As by PR #248, add-handle will should inject the handle's metadata in the collection. But further on we need to discuss, when and by what command line interface we want to update metadata in  a collection from the repositories of its handles indeed.